Gem/Code/Source/Components/NetworkSimplePlayerCameraComponent.cpp

```diff
@@ -72,13 +72,20 @@ namespace MultiplayerSample
         {
             const AZ::Quaternion targetRotation = AZ::Quaternion::CreateRotationZ(GetCameraYaw()) * AZ::Quaternion::CreateRotationX(GetCameraPitch());
             const AZ::Quaternion currentRotation = m_activeCameraEntity->GetTransform()->GetWorldTM().GetRotation();
-            const AZ::Quaternion aimRotation = m_syncOrientationImmediate ?
-                targetRotation : currentRotation.Slerp(targetRotation, cl_cameraBlendSpeed).GetNormalized();
+            AZ::Quaternion aimRotation;
+            if(m_syncOrientationImmediate)
```

Comment:
Are you seeing weird camera movements upon spawning otherwise?

Reply:
Yes, you'll see the camera slerp from origin to the intended initial orientation. It's honestly not a bad effect but shouldn't be default behavior.